client/my-sites/themes/theme-options.js

```diff
@@ -124,9 +131,13 @@ const preview = {
 	action: themePreview
 };
 
-const signupLabel = i18n.translate( 'Pick this design', {
-	comment: 'when signing up for a WordPress.com account with a selected theme'
-} );
+const signupLabel = ( state, siteId ) => config.isEnabled( 'jetpack/pijp' ) && isJetpackSite( state, siteId )
+	? i18n.translate( 'Upgrade to activate', {
+		comment: 'label prompting user to upgrade the Jetpack plan to activate a certain theme'
+	} )
```

Comment:
This can't be right -- `signupLabel` is meant to be displayed when a logged-out user goes to `/themes` and selects a theme to start a new blog with.